website/docs/r/dev_test_lab_schedule.html.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_dev_test_schedule"
+sidebar_current: "docs-azurerm-dev-test-lab-schedule"
+description: |-
+    Manages automated startup and shutdown schedules for Azure Dev Test Lab.
+---
+
+# azurerm_dev_test_schedule
+
+Manages automated startup and shutdown schedules for Azure Dev Test Lab.
+
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG"
+  location = "West US"
+}
+
+resource "azurerm_dev_test_lab" "test" {
+  name                = "acctestDTL"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+}
+
+resource "azurerm_dev_test_schedule" "test" {
+  name = "LabVmAutoStart"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  lab_name = "${azurerm_dev_test_lab.test.name}"
+  
+  weekly_recurrence {
+    time      = "1100"
+    week_days = ["Monday", "Tuesday"]
+  }
+
+  time_zone_id = "Pacific Standard Time"
+  task_type = "LabVmsStartupTask"
+
+  notification_settings {
+  }
+
+  tags = {
+    environment = "Production"
+  }
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the dev test lab schedule. Valid value for name
```

Comment:
We don't hard wrap lines like this anymore, could we remove the new lines and soft wrap all these paragraphs?

Reply:
Done.